services/storage-driver.js

```diff
@@ -62,14 +62,17 @@ module.exports = function (dbUrl, injection, appUtil) {
     service.findAll = function (table, filteringAndSorting) {
         return ensureIndexes(table, filteringAndSorting).then(function () {
             return Q(modelFor(table).find(queryFor(table, filteringAndSorting)).sort(sortingFor(filteringAndSorting, table.fields)).exec())
-                .then(function (result) { return result.map(fromBson(table)) });
+                .then(function (result) { return result.map(fromBson(table)); });
         });
     };
 
     service.findRange = function (table, filteringAndSorting, start, count) {
+        // Force numeric
+        start = Number(start);
+        count = Number(count);
```

Comment:
Well... The idea is OK, but what if someone will pass not a number here? It's better to make full check here. And doesn't driver has it's own protection against wrong arguments?

Reply:
Start&Count are pagination related, they will always be integers. With Node8 and updated moongose this crashes allcountjs as it always expects a number. Unfortunately driver doesn't do the conversion, it would just crash with error.